pkg/jobs/jobspb/jobs.go

```diff
@@ -44,3 +44,22 @@ func (fes RestoreFrontierEntries) Equal(fes2 RestoreFrontierEntries) bool {
 	}
 	return true
 }
+
+// RestoreFrontierEntries is a slice of RestoreFrontierEntries.
+// TODO(msbutler): use generics and combine with above.
+type ResolvedSpanEntries []ResolvedSpan
+
+func (fes ResolvedSpanEntries) Equal(fes2 ResolvedSpanEntries) bool {
```

Comment:
"fes" ?

Reply:
lol, i think i blindly copied from the method defined above it. Fixed.